docs/src/concepts.md

```diff
@@ -0,0 +1,116 @@
+| ElectrodeConfig | particular electrode configuration to be used for ephys recording. |
+| ElectrodeConfig.Electrode | corresponding electrodes in `ProbeType.Electrode` that are used for recording in this electrode configuration (e.g. for Neuropixels 1.0 or 2.0, there can be at most 384 electrodes usable for recording per probe). |
+
+### `ephys` schema ([API docs](../api/element_array_ephys/ephys))
+Tables related to information about physiological recordings and automatic ingestion of spike sorting results.
+
+| Table | Description |
+| --- | --- |
+| ProbeInsertion | A surgical insertion of a probe in the brain. Every experimental session consists of one or more entries in `ProbeInsertion` with a corresponding `InsertionLocation` each. |
+| EphysRecording | each `ProbeInsertion` is accompanied by a corresponding `EphysRecording`, specifying the `ElectrodeConfig` used for the recording from the `Probe` defined in `ProbeInsertion`. |
+| Clustering | specify instance(s) of clustering on an `EphysRecording`, by some `ClusteringMethod`. |
+| Curation | specify instance(s) of curations performed on the output of a given `Clustering`. |
+| CuratedClustering | set of results from a particular round of clustering/curation. |
+| CuratedClusting.Unit | Identified unit(s) from one `Curation`, and the associated properties (e.g. cluster quality, spike times, spike depths, etc.). |
+| WaveformSet | A set of spike waveforms for units from a given `CuratedClustering`. |
+
+## Element Development
+
+Through our interviews and direct collaboration on the precursor projects, we identified the common motifs to create the [Array Electrophysiology Element](https://github.com/datajoint/element-array-ephys).
+
+Major features of the Array Electrophysiology Element include:
+
++ Pipeline architecture detailed by:
+
+    + Probe, electrode configuration compatible with Neuropixels probes and generalizable to other types of probes (e.g. tetrodes) - supporting both `chronic` and `acute` probe insertion modes.
+
+    + Probe-insertion, ephys-recordings, LFP extraction, clusterings, curations, sorted units and the associated data (e.g. spikes, waveforms, etc.).
+    
+    + Store/track/manage different curations of the spike sorting results - supporting both curated clustering and kilosort triggered clustering (i.e., `no_curation`).
+
++ Ingestion support for data acquired with SpikeGLX and OpenEphys acquisition systems. 
++ Ingestion support for spike sorting outputs from Kilosort.
++ Triggering support for workflow integrated Kilosort processing.
++ Sample data and complete test suite for quality assurance.
+
+Incorporation of SpikeInterface into the Array Electrophysiology Element will be on DataJoint Elements development roadmap. Dr. Loren Frank has led a development effort of a DataJoint pipeline with SpikeInterface framework and NeurodataWithoutBorders format integrated (https://github.com/LorenFrankLab/nwb_datajoint).
```

Comment:
If we say this here, it should go in roadmap below

Reply:
Changed it, just pending commit.